### PR TITLE
107 repo urls in scenarios & 112 top level documentation

### LINKED ISSUE #107: Add support for github repository urls in scenarios for the testing pipeline
When a scenario is created, the inclusion of a `repo_url.txt` file should indicate that the seeding process in the pipeline needs to first download the code repository and then process it as if a `code.zip` was included in the scenario.

The `repo_url.txt` should have one line that is the url of the repository.

### LINKED ISSUE #112: Update top level documentation around testing

### PR DESCRIPTION
**Pull request by Sorrento110**
# Description

* Adds support for repo_url.txt files to define code to be processed in scenarios.
* Updates top level README for reporting test harness. 

Resolves #107 
Resolves #112

### CODE REVIEW

File: worker/utils.py [hunk 1]
```diff
@@ -252,7 +252,7 @@ def get_code_from_tds(code_id, code=False, dynamics_only=False):
     content_object = {}
 
     for name, blocks in file_names.items():
-        name = name.split("/")[-1]
+        # name = name.split("/")[-1]
```
Comment (brandomr) [inline worker/utils.py:255]: I'm curious what's going on with this commented out line?

File: reporting/scenarios/Penn CHIME/repo_url.txt
Comment (brandomr) [inline reporting/scenarios/Penn CHIME/repo_url.txt:1]: This scenario needs a description

**Review by brandomr — COMMENTED** [review]
minor comments--i will add the scenario description

**Review by brandomr — APPROVED** [review]

File: worker/utils.py (hunk 1 above)
Comment (Sorrento110) [reply worker/utils.py:255]: This was causing problems in repo's with deeper file trees. It was not able to find the correct files when fetching the code with this in place, we can just remove that line.

**Pull request merged**